package.json

```diff
@@ -10,18 +10,12 @@
   "bugs": "https://github.com/mauriciopoppe/function-plot/issues",
   "homepage": "http://mauriciopoppe.github.io/function-plot/",
   "author": "Mauricio Poppe <mauricio.poppe@gmail.com>",
-  "main": "dist/index.js",
+  "main": "dist/function-plot.js",
```

Comment:
I think that if an app imports the library it should import the main non-bundled file i.e. the library entrypoint

Reply:
This PR was some time ago, so I don't know exactly what I did here, but index.js isn't even in the build output. `function-plot.js` (as specified in webpack.config.json) is. Code splitting is not a problem here, since there is only one entrypoint.

Reply:
The build process is:
- tsc - which generates dist/index.js
- webpack - which bundles all the ts transpiled files into a single minified file

An app that imports this project as a dependency should include dist/index.js IMO